core/src/main/java/com/novoda/landingstrip/Notifier.java

```diff
@@ -25,6 +28,13 @@ private void recreateAndBindTabs(BaseAdapter<T> adapter) {
         }
     }
 
+    private void handleAdapterSetBecausePageSelectedIsNotCalled(BaseAdapter<T> adapter) {
```

Comment:
should change the method name here - the ScrollingPageChangeListener knows about PageSelected since it implements OnPageChangeListener. This class doesn't.

How about simply `updateActivatedTab()` or `syncActivatedTab`?